plat/ti/k3/include/platform_gic.h

```diff
@@ -0,0 +1,16 @@
+/*
```

Comment:
Because we have a global INCLUDES namespace for all the headers, the platform specific headers within the global INCLUDES should have a specific prefix, say  `plat_k3_*`. 

I am less concerned about the .c file names though.

Reply:
No other platform does this, is this something new?

Reply:
The headers expected to be exported to the `INCLUDES` from the platform with generic names are platform_def.h, plat_ld.S (if required) and plat_macros.S. Other headers are supposed to have a platform specific name. ARM platforms do this and most of the platforms do this as well, although there are a few exceptions. This is one of things I am looking out for as the number of platforms upstream increase unless we can namespace the header inclusion based on the directory.

Reply:
Okay, thanks for the explanation, will fix.